other/install_2-1_mysql.sql

```diff
@@ -807,9 +807,8 @@ CREATE TABLE {$db_prefix}messages (
 	approved TINYINT NOT NULL DEFAULT '1',
 	likes SMALLINT UNSIGNED NOT NULL DEFAULT '0',
 	PRIMARY KEY (id_msg),
-	UNIQUE idx_id_board (id_board, id_msg),
+	UNIQUE idx_id_board (id_board, id_msg, approved),
```

Comment:
Must reorder the columns low cardinality first so that it  gets this instead of `idx_approved`
```suggestion
	UNIQUE idx_id_board (approved, id_board, id_msg),
```

Reply:
I suspect your 2s measurement was an outlier; I would repeat that a few times & take an average.  On my *3M* row db on localhost, that query averages 0.06592s.  On my 200k row db on localhost, that query averages .01784s.  

Changing the order as you recommended above increased the average on the 3M row db to .07108, a decrease in performance.  

A good rule of thumb is to choose the columns as their most restrictive first, which in this case, is id_board.  That helps narrow the selection more quickly.  A boolean value will not help you do that so much:
https://www.percona.com/blog/2009/06/05/a-rule-of-thumb-for-choosing-column-order-in-indexes/

Reply:
I suspect your 2s measurement was an outlier; I would repeat that a few times & take an average.  On my *3M* row db on localhost, that query averages 0.06592s.  On my 200k row db on localhost, that query averages .01784s.  

Changing the order as you recommended above increased the average on the 3M row db to .07108, a decrease in performance.  

A good rule of thumb is to choose the columns as their most restrictive first, which in this case, is id_board.  That helps narrow the selection more quickly.  A boolean value will not help you do that so much:
https://www.percona.com/blog/2009/06/05/a-rule-of-thumb-for-choosing-column-order-in-indexes/